Spatial.gd

```diff
@@ -119,12 +119,13 @@ func exit_burrito():
 
 
 func set_minimal_mouse_block():
-	var top_corner := Vector2(287, 0)
-	var bottom_corner := Vector2(314, 32)
+	var menu_button = $Control/GlobalMenuButton
+	var top_corner = menu_button.get_position()
+	var bottom_corner = menu_button.get_position() + menu_button.get_size()
```

Comment:
By changing the values here to the properties the numbers come from, any more changes to the icons will be updated in the passthrough